cibuildwheel/platforms/ios.py

```diff
@@ -598,6 +598,26 @@ def build(options: Options, tmp_path: Path) -> None:
 
                     log.step("Running test suite...")
 
+                    # iOS doesn't support placeholders in the test command,
+                    # because the source dir isn't visible on the simulator.
+                    if (
+                        "{project}" in build_options.test_command
+                        or "{package}" in build_options.test_command
+                    ):
+                        msg = unwrap_preserving_paragraphs(
+                            f"""
+                            iOS tests configured with a test command that uses the {{"project"}} or
```

Comment:
Are the quotes supposed to be on the inside? `{"project"}` vs. `"{project}"`?

Reply:
Good catch!